.github/workflows/test.yaml

```diff
@@ -266,6 +266,53 @@ jobs:
           max_attempts: 3
           retry_wait_seconds: 30
           command: rad deploy ${{ matrix.path }} ${{ matrix.deployArgs }}
+      - name: Wait for all pods to be ready
+        if: steps.gen-id.outputs.RUN_TEST == 'true'
+        id: wait-for-pods
+        run: |
+          namespace="${{ matrix.env }}-${{ matrix.app }}"
+          label="radapp.io/application=${{ matrix.app }}"
+          kubectl rollout status deployment -l $label -n $namespace --timeout=90s
```

Comment:
We don't need this because `rad deploy` already waits for all pods to be ready. We discussed this with @rynowak.

Reply:
Ideally we don't need this 😉. If we're inserting waits in random places it can mask bugs.

Reply:
yes, i had seen the comment in an earlier PR where this isn't needed. I was curious what the output of this would be now since the endpoint isn't currently loading. no intention to merge this in.